train-task.py

```diff
@@ -193,26 +224,32 @@ def train(self, output_dir, train_dataset, eval_dataset, logger, device):
         column_names = train_dataset.column_names
         model = self.pretrained_model.to(self.device)
 
-        train_dataset_samsum_pt = train_dataset.map(self.convert_examples_to_features, batched=True,
-                                                   remove_columns=column_names)
+        train_dataset_samsum_pt = train_dataset.map(
+            self.convert_examples_to_features,
+            batched=True,
+            remove_columns=column_names,
+        )
 
-        eval_dataset_samsum_pt = eval_dataset.map(self.convert_examples_to_features, batched=True,
-                                                  remove_columns=column_names)
+        eval_dataset_samsum_pt = eval_dataset.map(
+            self.convert_examples_to_features,
+            batched=True,
+            remove_columns=column_names,
+        )
 
         # train_dataset_samsum_pt = train_dataset_samsum_pt.shard(num_shards=50, index=0) # Cut part of the train dataset to speed up testing
         # eval_dataset_samsum_pt = eval_dataset_samsum_pt.shard(num_shards=50, index=0) # Cut part of the train dataset to speed up testing
 
-        seq2seq_data_collator = DataCollatorForSeq2Seq(self.tokenizer, model=model,
-                                                       pad_to_multiple_of=8)
+        seq2seq_data_collator = DataCollatorForSeq2Seq(self.tokenizer, model=model, pad_to_multiple_of=8)
 
         train_dataloader, batch_size = self.partition_dataset(train_dataset_samsum_pt, seq2seq_data_collator)
```

Comment:
@SofiaChar Is this `batch_size` unused on purpose?

Reply:
The batch size for the training is calculated in the partition dataset. The batch_size is there for running on the test dataset. Maybe I delete the whole testing since I don’t use the test data anymore and  `calculate_metric_on_test_ds` is not used.

Reply:
Not sure I'm 100% following – this `batch_size` is assigned here, but is not used on the following line (where it says `batch_size=1`).

Reply:
This was resolved in #2.